iowrappers/photos_client.go

```diff
@@ -23,6 +23,10 @@ var isHtmlAnchor = func(node *html.Node) bool {
 	return node.Type == html.ElementNode && node.Data == "a"
 }
 
+var isValidPhotoUrl = func(url string) bool {
+	return strings.HasPrefix(url, "https://lh3.googleusercontent.com/places/")
```

Comment:
Nit: put the url in a const

Reply:
It will give a compilation error when changed to const. I am not sure how function object is treated in golang.